### PR TITLE
remove use_lazy_graph_module flag

### PR DESCRIPTION
**Pull request by bobrenjc93**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #136962

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @rec

### CODE REVIEW

File: torch/_dynamo/convert_frame.py [hunk 1]
```diff
@@ -820,9 +820,7 @@ def count_args(code: CodeType) -> int:
 
         return guarded_code
 
-    with _use_lazy_graph_module(config.use_lazy_graph_module), compile_context(
-        CompileContext(compile_id)
-    ):
+    with _use_lazy_graph_module(True), compile_context(CompileContext(compile_id)):
```
Comment (anijain2305) [inline torch/_dynamo/convert_frame.py:823]: We should probably remove this decorator as well?

**Comment by anijain2305** [comment]
cc @shunting314 . We are removing already established config flags. This one has repercussions throughout torch.compile stack, so extra set of eyes would be good.

**Comment by bobrenjc93** [comment]
This ended up being a bit more nontrivial than I expected. I'll put this back in the queue for now in case someone else wants to take it.

**Pull request closed without merging**